### PR TITLE
New version: winpython.winpython version 4.6.20220116

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/43920)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
The `InstallerRegex` needs to be fixed, will do it tomorrow. (https://github.com/microsoft/winget-pkgs/pull/43921, https://github.com/microsoft/winget-pkgs/pull/43922)

File: manifests/w/winpython/winpython/4.6.20220116/winpython.winpython.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+InstallerType: exe
+InstallerSwitches:
+  Silent: -y -o".\winpython"
+  SilentWithProgress: -y -o".\winpython"
+FileExtensions:
+- py
+- pyc
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/winpython/winpython/releases/download/4.6.20220116/Winpython32-3.9.10.0dot.exe
```
Comment (ItzLevvie) [inline manifests/w/winpython/winpython/4.6.20220116/winpython.winpython.installer.yaml:16]:
```suggestion
  InstallerUrl: https://github.com/winpython/winpython/releases/download/4.6.20220116/Winpython64-3.10.2.0.exe
```

File: manifests/w/winpython/winpython/4.6.20220116/winpython.winpython.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.0.6 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: winpython.winpython
+PackageVersion: 4.6.20220116
```
Comment (ItzLevvie) [inline manifests/w/winpython/winpython/4.6.20220116/winpython.winpython.installer.yaml:5]:
```suggestion
PackageVersion: 3.10.2.0
```

File: manifests/w/winpython/winpython/4.6.20220116/winpython.winpython.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.0.6 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: winpython.winpython
+PackageVersion: 4.6.20220116
```
Comment (ItzLevvie) [inline manifests/w/winpython/winpython/4.6.20220116/winpython.winpython.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.10.2.0
```

File: manifests/w/winpython/winpython/4.6.20220116/winpython.winpython.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.6 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: winpython.winpython
+PackageVersion: 4.6.20220116
```
Comment (ItzLevvie) [inline manifests/w/winpython/winpython/4.6.20220116/winpython.winpython.yaml:5]:
```suggestion
PackageVersion: 3.10.2.0
```

File: manifests/w/winpython/winpython/4.6.20220116/winpython.winpython.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+InstallerSwitches:
+  Silent: -y -o".\winpython"
+  SilentWithProgress: -y -o".\winpython"
+FileExtensions:
+- py
+- pyc
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/winpython/winpython/releases/download/4.6.20220116/Winpython32-3.9.10.0dot.exe
+  InstallerSha256: 497F9B4D1D3BEBC3606C94911673B843DDA3F0B226876B7240982E3E874BB52F
```
Comment (ItzLevvie) [inline manifests/w/winpython/winpython/4.6.20220116/winpython.winpython.installer.yaml:17]:
```suggestion
  InstallerSha256: 26F057CCBAD8B9DD5B42A18BA2DC8D3A8C2A8A9CBD6D199239E956D361540790
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**